misc/python/materialize/mzcompose.py

```diff
@@ -785,7 +787,7 @@ def __init__(
             "MZ_LOG_FILTER",
             "MZ_SOFT_ASSERTIONS=1",
         ],
-        volumes: List[str] = [],
```

Comment:
I think we don't want `dataflowd` to have access to the catalog directory. Ideally, it shouldn't do any local disc I/O at the moment.

Reply:
Aren't we doing CSV file ingestion via the dataflow workers? I was getting test failures if I did not give dataflowd disk access to /tmp ?

Reply:
You're correct, but it's probably not the architecture we'd want to have in the long run. I'm ok with giving dataflowd read access to /tmp, but please add a comment that this is not intended to stay. Or we disable the csv tests for nightly...

Reply:
Comment added. Thank you.